channels/generic/http.py

```diff
@@ -1,5 +1,7 @@
-from channels.consumer import AsyncConsumer
+from _ast import Dict
```

Comment:
Is there no way to avoid this import? :-(

Reply:
Yes, it was a mistake, it should be
```python
from typing import Dict
```

I should have fixed it